libs/types/src/cdf/ballot-definition/convert.ts

```diff
@@ -99,13 +98,15 @@ export function convertVxfElectionToCdfBallotDefinition(
                   ),
                   // Create write-in options up to the number of votes allowed
                   ...(contest.allowWriteIns
-                    ? take(contest.seats, naturals()).map(
-                        (writeInIndex): Cdf.CandidateOption => ({
-                          '@type': 'BallotDefinition.CandidateOption',
-                          '@id': `${contest.id}-option-write-in-${writeInIndex}`,
-                          IsWriteIn: true,
-                        })
-                      )
+                    ? naturals()
+                        .take(contest.seats)
```

Comment:
When I originally wrote this code, I wanted to just write `range(0, contest.seats)` and get an array. But I just worked with what we had.

Sometimes it takes a little more mental gymnastics to work in infinite sequence land when you're trying to do a finite thing. I wonder if we could have some finite versions of all these awesome infinite sequence tools.

Reply:
It could also be `integers({ from: 1, through: contest.seats }).toArray()`, which would yield the same results (`naturals()` is just `integers({ from: 1 })`). It sounds like you wanted this to be 0-indexed, though. That'd be `integers({ through: contest.seats - 1 }).toArray()`.